pandas/core/format.py

```diff
@@ -248,18 +253,13 @@ def to_string(self):
 
 
 def _strlen_func():
-    if compat.PY3:  # pragma: no cover
-        _strlen = len
+    use_east_asian_width = get_option("display.unicode.east_asian_width")
+    encoding = get_option("display.encoding")
```

Comment:
ahh I see you have the function I described above, simply make this a function, then pass directly in